test-suite/Makefile.am

```diff
@@ -124,23 +124,27 @@ squid-conf-tests: $(srcdir)/test-squid-conf.sh $(top_builddir)/src/squid.conf.de
 			exit 1; \
 		fi; \
 	done; \
-	failed=0; \
+	result=0; \
 	cfglist="$(top_builddir)/src/squid.conf.default $(srcdir)/squidconf/*.conf"; \
 	rm -f $@ || $(TRUE); \
 	for cfg in $$cfglist ; do \
-		$(srcdir)/test-squid-conf.sh $(top_builddir) $(sbindir) $$cfg || \
-			{ echo "FAIL: squid.conf test: $$cfg" | \
+		$(srcdir)/test-squid-conf.sh $(top_builddir) $(sbindir) $$cfg; result=$$?; \
+        case $$result in \
+        0) echo "PASS: squid.conf test: $$cfg" | \
 				sed s%$(top_builddir)/src/%% | \
 				sed s%$(srcdir)/squidconf/%% ; \
-				failed=1; break; \
-			}; \
-		if test "$$failed" -eq 0; then \
-			echo "PASS: squid.conf test: $$cfg" | \
+                continue ;; \
+        77) echo "SKIP: squid.conf test: $$cfg" | \
```

Comment:
I wonder what it would take to stop emulating automake test driver here and actually reuse it? Do you know?

Reply:
> I wonder what it would take to stop emulating automake test driver here and actually reuse it? Do you know?

From what I know, each test would have to be its own script, not an iteration in a for loop in Makefile.am

Reply:
Hm... We are able to use cfgaux/test-driver to testHeaders. These Squid configuration tests feel rather similar:

```
testHeaders: $(SOURCES) $(noinst_HEADERS) $(EXTRA_DIST) ...

squid-conf-tests: ... $(srcdir)/squidconf/*
```

With a few adjustments, we may even be able to move to something like this:

```diff
- squid-conf-tests: ... $(srcdir)/squidconf/*
+ squid-conf-tests: ... $(EXTRA_DIST)
```

The current squid-conf-tests script gets more complex with every modification, and we are writing it in a Makefile! Do you think the above conversion is worth trying?